### PR TITLE
small ui adjustments

### PR DESCRIPTION
**Pull request by Niklap97**
- make nk3 button scalable
- center the messageboxes accordingly
- center logo
- remove stylesheet colors

### CODE REVIEW

**Review by daringer — APPROVED** [review]

File: nitrokeyapp/update.py [hunk 1]
```diff
@@ -49,6 +49,7 @@ def __init__(
         self.bar_download = overview_tab.ui.progressBar_Download
         self.bar_finalization = overview_tab.ui.progressBar_Finalization
         self.info_frame = info_frame
+        self.parent = overview_tab.parent_ui
```
Comment (robin-nitrokey) [inline nitrokeyapp/update.py:52]: Can’t we just use `overview_tab`?

**Comment by robin-nitrokey** [comment]
The button is still truncated for me (even with a bigger window size):

app

**Comment by Niklap97** [comment]
did you rebuild the ui files with `make build-ui` ?

**Comment by robin-nitrokey** [comment]
> did you rebuild the ui files with `make build-ui` ?

I must have missed that.  Looks good!

**Review by robin-nitrokey — CHANGES_REQUESTED** [review]
Please add `qt_material` to the mypy whitelist to fix the CI.

**Comment by robin-nitrokey** [comment]
I think the difference between enabled and disabled icons is a bit too subtle with the default settings, but we can address that later.

**Review by robin-nitrokey — APPROVED** [review]

**Pull request merged**